client-reactjs/src/components/admin/Consumers.js

```diff
@@ -177,7 +177,7 @@ class Consumers extends Component {
   }
 
   searchADGroups(searchString) {
-    if(searchString.length <= 2)
+    if(searchString.length <= 0)
```

Comment:
any reason why this condition was removed?

Reply:
This is a mistake. I reverted it back